src/components/Table/Table.types.part.tsx

```diff
@@ -29,6 +29,22 @@ export interface TableCellEvent<T> {
   data?: T;
 }
 
+export interface TableSortEvent {
```

Comment:
You have to export this type in [`src/components/Table/index.tsx`](https://github.com/ZEISS/precise-ui/blob/86be817ae4e6ca08a996f941309126851c79d0b1/src/components/Table/index.tsx#L14)